src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -6,74 +6,30 @@ import com.intellij.openapi.project.Project
 import com.intellij.ui.components.JBLabel
 import com.intellij.util.concurrency.annotations.RequiresEdt
 import com.intellij.util.ui.UIUtil
-import com.sourcegraph.cody.chat.SignInWithSourcegraphPanel
-import com.sourcegraph.cody.chat.ui.CodyOnboardingGuidancePanel
-import com.sourcegraph.cody.config.CodyAccount
-import com.sourcegraph.cody.config.CodyApplicationSettings
-import com.sourcegraph.cody.config.CodyAuthenticationManager
 import com.sourcegraph.cody.ui.web.WebUIService
 import java.awt.CardLayout
 import java.awt.GridBagConstraints
 import java.awt.GridBagLayout
-import java.awt.GridLayout
 import javax.swing.JComponent
 import javax.swing.JPanel
 
 @Service(Service.Level.PROJECT)
 class CodyToolWindowContent(project: Project) {
-  private val cardLayout = CardLayout()
-  private val cardPanel = JPanel(cardLayout)
-  val allContentPanel: JComponent = JPanel(GridLayout(1, 1))
+  val allContentPanel: JComponent = JPanel(CardLayout())
+  private val mainPanel = JPanel(GridBagLayout())
```

Comment:
We were using the cardlayout to swap between the loading, login and onboarding components.

We can't put the webview in a CardLayout because JetBrains remote doesn't render it, so we are swapping the components.

But now there are only two components to swap: Loading and webview. So we don't need this panel anymore.

 Let's just have `allContentPanel` with whatever trivial layout (GridLayout(1,1) etc.)

Reply:
So this is weird, because I tried to simplify it at the start but it was not working without `CardLayout`.
But maybe it was error on my side, I will check again :)